tests/ceph_ansible/purge_dashboard.py

```diff
@@ -0,0 +1,56 @@
+""" Module to purge ceph dashboard"""
+import logging
+import re
+
+log = logging.getLogger(__name__)
+
+
+def run(**kw):
```

Comment:
please check other test modules, Hope you may need ceph_cluster to point which exact cluster in case multiple ceph clusters.

Reply:
ceph cluster nodes we can get it from **kw itself, https://github.com/red-hat-storage/cephci/blob/31922863cb5cf994cd27a6427174d734156cf81c/tests/ceph_ansible/purge_cluster.py#L28
So i don't think we need ceph_cluster also as param.